apps/src/templates/teacherDashboard/TeacherSectionSelectorMenuItem.jsx

```diff
@@ -28,13 +32,12 @@ export default class TeacherSectionSelectorMenuItem extends Component {
   render() {
     const {section, onClick} = this.props;
     return (
-      <PopUpMenu.Item onClick={onClick}>
-        <div style={styles.item}>
-          <span>{section.name}</span>
-          <span style={styles.assigned}>
-            {section.isAssigned && <FontAwesome icon="check" />}
-          </span>
-        </div>
+      <PopUpMenu.Item onClick={onClick} style={styles.item}>
+        <span style={styles.assigned}>
+          {section.isAssigned && <FontAwesome icon="check" />}
+          {!section.isAssigned && <span>&nbsp;&nbsp;&nbsp;&nbsp;</span>}
```

Comment:
this is not the best way to ensure the component retains its width. can you try adding a `width` property to `styles.assigned` instead? if no easy solution emerges, please at least make sure the alignment doesn't look too crazy when you zoom in and out.

Reply:
This did feel a little hacky. I tried to set width and it didn't work.  I also tried always a checkmark there to hold the spot - just making it white when not assigned, but since we have a grey overlay on hover that didn't work either. Any other ideas? I'll check zoomed in/out...

Reply:
If I zoom way out: 
<img width="767" alt="Screen Shot 2019-11-20 at 1 09 35 PM" src="https://user-images.githubusercontent.com/12300669/69278390-15af5200-0b97-11ea-90c1-de54eb5c4c52.png">
Or zoom way in: 
<img width="581" alt="Screen Shot 2019-11-20 at 1 09 52 PM" src="https://user-images.githubusercontent.com/12300669/69278396-1a740600-0b97-11ea-8d94-537fc807688d.png">

my space hack holds up just fine, but the anchor points for the pop-up-menu are messed up. 

This is true for other uses of the PopUpMenu too: 
<img width="537" alt="Screen Shot 2019-11-20 at 1 07 48 PM" src="https://user-images.githubusercontent.com/12300669/69278456-3aa3c500-0b97-11ea-9672-dbd3d28ff558.png">

How much of an issue do we think this is? Is there any way to know frequency with which users zoom on our pages? Doesn't feel blocking to me for this change, but I'll add anchor points for pop up menus to our Screen Resolution epic.

Reply:
https://codedotorg.atlassian.net/browse/LP-1047

Reply:
thanks for testing that out! can you try one more thing? `visibility: hidden` if that doesn't work, then fine to merge what you have.

Reply:
visibility works - thanks!